var/spack/repos/builtin/packages/hipblas/package.py

```diff
@@ -159,8 +160,9 @@ def cmake_args(self):
 
         if self.spec.satisfies("^cmake@3.21.0:3.21.2"):
             args.append(self.define("__skip_rocmclang", "ON"))
-
-        if self.spec.satisfies("@5.2.0:"):
+        if self.spec.satisfies("@:5.1.3"):
+            args.append("-DCMAKE_MODULE_PATH={0}/".format(self.spec["hip"].prefix.cmake))
```

Comment:
Same question on `/` as above

Reply:
Made the changes as per the review comments.